__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET/api/topics", () => {
+    it("Response has to be 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+})
+
+it("Responds with the correct keys", () => {
+    return request(app)
```

Comment:
This single test would do the job as it checks the status, therefore the test above was unnecessary